[2025-06-11T09:34:21.537+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-11T09:34:21.559+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.fetch_and_store_covid_data manual__2025-06-11T09:34:20.502065+00:00 [queued]>
[2025-06-11T09:34:21.569+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.fetch_and_store_covid_data manual__2025-06-11T09:34:20.502065+00:00 [queued]>
[2025-06-11T09:34:21.569+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-06-11T09:34:21.584+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_and_store_covid_data> on 2025-06-11 09:34:20.502065+00:00
[2025-06-11T09:34:21.592+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=372) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-11T09:34:21.594+0000] {standard_task_runner.py:63} INFO - Started process 373 to run task
[2025-06-11T09:34:21.596+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'fetch_and_store_covid_data', 'manual__2025-06-11T09:34:20.502065+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/ covid_pipeline.py', '--cfg-path', '/tmp/tmpt2dwj6e4']
[2025-06-11T09:34:21.598+0000] {standard_task_runner.py:91} INFO - Job 52: Subtask fetch_and_store_covid_data
[2025-06-11T09:34:21.654+0000] {task_command.py:426} INFO - Running <TaskInstance: covid_data_pipeline.fetch_and_store_covid_data manual__2025-06-11T09:34:20.502065+00:00 [running]> on host da27caf1d70a
[2025-06-11T09:34:21.751+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_and_store_covid_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-11T09:34:20.502065+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-11T09:34:20.502065+00:00'
[2025-06-11T09:34:21.751+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-11T09:34:22.236+0000] {fetch_and_store.py:19} INFO - Fetched data: {'cases': 704753890, 'deaths': 7010681, 'recovered': 675619811, 'timestamp': datetime.datetime(2025, 6, 11, 9, 34, 22, 236330)}
[2025-06-11T09:34:22.241+0000] {fetch_and_store.py:53} INFO - ⏭️ No new data. Latest record matches current data. Skipping insert.
[2025-06-11T09:34:22.245+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-11T09:34:22.247+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-11T09:34:22.268+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=fetch_and_store_covid_data, run_id=manual__2025-06-11T09:34:20.502065+00:00, execution_date=20250611T093420, start_date=20250611T093421, end_date=20250611T093422
[2025-06-11T09:34:22.329+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-11T09:34:22.355+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-11T09:34:22.356+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
